client/ayon_core/hosts/max/api/pipeline.py

```diff
@@ -240,10 +240,9 @@ def get_previous_loaded_object(container: str):
         node_list(list): list of nodes which are previously loaded
     """
     node_list = []
-    sel_list = rt.getProperty(container.modifiers[0].openPypeData, "sel_list")
-    for obj in rt.Objects:
-        if str(obj) in sel_list:
-            node_list.append(obj)
+    node_transform_monitor_list = rt.getProperty(container.modifiers[0].openPypeData, "all_handles")
```

Comment:
Pretty sure this exceeds the allowed styling line length.
```suggestion
    node_transform_monitor_list = rt.getProperty(
        container.modifiers[0].openPypeData, "all_handles")
```